### DISCUSSION TITLE
How to define column label in prisma ?

### DISCUSSION THREAD

**Discussion by xiangnanscu** [prisma/prisma#15501] [Q&A] [answered]
Hello, When I define a database column name (in English) in prisma, usually there's a label (in Chinese) for it. I need prisma to record this info and then can access from prisma client api. I'm wondering if there's something  like `@label` syntax in prisma.
```
model User {
  id         Int      @id @default(autoincrement())
  name       String   @label("姓名")
  permission Int      @default(0) @label("权限")
}
```
And then `prisma.user.labels` will be `['id','姓名','权限']`. Or Is there something similar to achieve this gloal?

**Comment by janpio** [comment, marked as answer]
Right now no such mechanism exists. You want to use `permission` in your database and Prisma Client API, but then be able to get a non English label for it?

**Reply by xiangnanscu** [reply]
Yes you're right

**Reply by pantharshit00** [reply]
Hello @xiangnanscu 

I will recommend you to open a feature request here about this: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Reply by xiangnanscu** [reply]
> Hello @xiangnanscu
> 
> I will recommend you to open a feature request here about this: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

thanks baby, I submitted it: https://github.com/prisma/prisma/issues/16129

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
